.projenrc.js

```diff
@@ -171,11 +172,19 @@ project.eslint.addOverride({
     "plugin:react-hooks/recommended",
     "plugin:jsx-a11y/recommended",
   ],
-  plugins: ["jsx-a11y"],
+  plugins: ["jsx-a11y", "prefer-arrow"],
   rules: {
     "react/jsx-sort-props": ["warn"],
     "react/react-in-jsx-scope": ["off"],
     "react/prop-types": ["off"],
+    "no-use-before-define": "off",
```

Comment:
I'm just being explicit about disabling this specific version of the rule as it will report incorrect errors in typescript. The one we actually want is the `@typescript-eslint/no-use-before-define` which is set below. [Reference](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-use-before-define.md#how-to-use)